packages/angular-table/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@tanstack/angular-table",
+  "version": "8.12.0",
+  "description": "Headless UI for building powerful tables & datagrids for Angular.",
+  "author": "Tanner Linsley",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/TanStack/table.git",
+    "directory": "packages/angular-table"
+  },
+  "homepage": "https://tanstack.com/table",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "keywords": [
+    "angular",
+    "table",
+    "angular-table",
+    "datagrid"
+  ],
+  "type": "module",
+  "module": "build/lib/esm2022/index.mjs",
+  "main": "build/lib/index.d.ts",
+  "types": "build/lib/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./build/lib/index.d.ts",
+      "import": "./build/lib/esm2022/index.mjs",
+      "default": "./build/lib/index.d.ts"
+    },
+    "./package.json": {
+      "default": "./package.json"
+    }
+  },
+  "engines": {
+    "node": ">=12"
+  },
+  "files": [
+    "build/lib/*",
+    "src"
+  ],
+  "scripts": {
+    "clean": "rimraf ./build",
+    "test:types": "tsc --noEmit",
+    "build": "pnpm ng-packagr -p ng-package.json",
+    "build:types": "tsc --emitDeclarationOnly"
+  },
+  "dependencies": {
+    "tslib": "^2.3.0",
```

Comment:
Why is tslib a package dependency? Maybe this should just be in the root package.json? Or a devDep here?

Reply:
Yes we can move it root package.json